### PR TITLE
Added fun cog function "emojify", fixed "react" command

### PR DESCRIPTION
**Pull request by AnonGuy**
I've defined a new `Bot` class that inherits from `discord.Bot`. This will allow us to define custom methods that can be used globally via the `Bot` instance (`bot`).

To celebrate this new era of Discord prosperity, I've implemented a `bot.emojify` method, which is a method that takes a string and returns a list of emoji substitutions for each alphabetic character given.

This will be particularly useful for `fun` cog reactions, such as #48 and #47.

### CODE REVIEW

**Comment by AnonGuy** [comment]
The mighty Travis has accepted my offering. It is known.

**Comment by AnonGuy** [comment]
@thebeanogamer I've selectively ignored a Codacy issue that popped up for this PR:

image

In this context, the non-referenced class contributes to a positive code style overall, because
* It prevents the need for an extra import in every bot cog where it will be used
* It allows any bot cog to be able to use this method by referencing the `bot` instance
* It allows other contributors to add useful methods and access them using the same format 

In my opinion, following PyLint for this specific use-case would make things less readable.

**Comment by AnonGuy** [comment]
I'm open to persuasion for this one, though. Feel free to enable that particular rule if you disagree with that opinion.

Also, @shadycake thoughts?

File: bot/__main__.py [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
 admins = []
 
-bot = Bot(
+ascii_lowercase += ' '
+
+
+class CustomBot(Bot):
+    """Bot class with custom methods."""
+
+    def emojify(self, message: str) -> List[str]:
```
[the remark this answers is not in the document]
Comment (shadycake) [reply bot/__main__.py]: @AnonGuy I get what you say about keeping it in here for easy access, but surely then you could just make it a `@staticmethod`?

**Comment by fractalpenguin** [comment]
@AnonGuy, why is there a need to implement a custom bot class, when the method can be implemented in the Fun cog? There seems no need to subclass the default Bot class.

File: bot/__main__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnonGuy) [reply bot/__main__.py]: That's a good point, actually, I didn't think that static methods are available to instances as well as classes.
Will implement this soon.

**Comment by fractalpenguin** [comment]
@AnonGuy, could you move the function to the Fun cog? I don't see the need to subclass Bot.

**Comment by AnonGuy** [comment]
@fractalpenguin The idea for having a `CustomBot` class is so that the methods can be accessed from any cog, since all cogs have access to the bot instance. Any cog will be able to use `self.bot.emojify`, for example. It also allows us to define new methods that we can access from outside the fun cog in the future, without the need for imports.

**Comment by fractalpenguin** [comment]
Methods which use the emojify function will be located in the Fun cog.

**Comment by shadycake** [comment]
@fractalpenguin Does have a point, there isn't any reason for emojify to ever be used outside of the Fun cog.

**Comment by AnonGuy** [comment]
Hmm, okay, I'll move the static method over to the Fun cog

**Comment by AnonGuy** [comment]
I've fully tested all emojis on a local bot instance, and implemented a working `!react` command.

**Comment by teamshortcut** [comment]
WELP SORRY BEANO IF I'M NOT MEANT TO DO THIS

**Pull request merged**